### PR TITLE
[Ruler] Change ruler scale down period from 60s to 600s

### PR DESCRIPTION
**Pull request by ying-jeanne**
#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@
 * [CHANGE] Compactor: the `/api/v1/upload/block/{block}/finish` endpoint now returns a `429` status code when the compactor has reached the limit specified by `-compactor.max-block-upload-validation-concurrency`. #4598
 * [CHANGE] Compactor: when starting a block upload the maximum byte size of the block metadata provided in the request body is now limited to 1 MiB. If this limit is exceeded a `413` status code is returned. #4683
 * [CHANGE] Store-gateway: cache key format for expanded postings has changed. This will invalidate the expanded postings in the index cache when deployed. #4667
+* [CHANGE] Ruler: changed ruler autoscaling policy, extended scale down period from 60s to 600s. #4786
```
Comment (aknuds1) [inline CHANGELOG.md:36]: This needs to be moved to the "main / unreleased" section.

**Review by aknuds1 — DISMISSED** [review]
The changelog item needs to move.

File: CHANGELOG.md (hunk 1 above)
Comment (pracucci) [reply CHANGELOG.md]: Thanks for spotting it!

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Ruler: changed ruler autoscaling policy, extended scale down period from 60s to 600s. #4786
```
Comment (pracucci) [inline CHANGELOG.md:7]: And should be moved under the `### Jsonnet` section, please.

File: operations/mimir/autoscaling.libsonnet [hunk 3]
```diff
@@ -53,6 +53,11 @@
   local replicasWithWeight(replicas, weight) =
     if weight <= 0.5 then std.ceil(replicas * weight) else std.floor(replicas * weight),
 
+  // getScaleDownPeriod will return the scale down period expressed in seconds.
```
Comment (aknuds1) [inline operations/mimir/autoscaling.libsonnet:56]: [Nit] I think "returns" is more standard verbiage for function documentation, and more explicit for the reader:
```suggestion
  // getScaleDownPeriod returns the scale down period expressed in seconds.
```

**Review by pracucci — APPROVED** [review]

**Review by aknuds1 — APPROVED** [review]
LGTM, but see nit.

**Pull request merged**